src/dotnet/CarbonAware.WebApi/api.yaml

```diff
@@ -127,8 +127,16 @@ components:
     CarbonAwareError:
       type: object
       properties:
-        message:
+        type:
```

Comment:
I think the fact that we have to update this _artisanal hand-crafted API spec_ is perhaps an indicator that we should really prioritize that auto-generation of the spec story that's sitting out there on the backlog